app/utils/__init__.py

```diff
@@ -1,5 +1,7 @@
 from __future__ import annotations
 
+import asyncio
+import contextlib
```

Comment:
We import with `from` everywhere else.

```suggestion
from contextlib import suppress
```